Update Readme (#84)
CorpGlory/grafana-progress-list: README.md

A panel showing list of progress-like items in one board. More about [development of the plugin](https://corpglory.com/s/grafana-progress-list/).

## Demo

See the [Demo](https://grafana.corpglory.com/d/2v8-HypGk/progress-list-01-basic?orgId=4) 

## How To Use

> 1. Create a metric query with table format data

Reviewer: ```suggestion
1. Create a query with Table data format
```

> 2. Go to "Options" tab and select `key` columns for mapping values.

Reviewer: ```suggestion
2. Go to "Options" tab and select which column to use as bars title and which columns to skip
```

> 3. Customise Progress List by using options.

Reviewer: ```suggestion
3. Customize Progress List appearance by using other options
```

> More info in [**tutorials**](https://github.com/CorpGlory/grafana-progress-list/wiki)

Reviewer: ```suggestion
More info in [**wiki**](https://github.com/CorpGlory/grafana-progress-list/wiki)
```
